src/routes/admin/+page.svelte

```diff
@@ -23,6 +25,33 @@
     newOrgSeats = 0;
     showAddOrganizationModal = false;
   };
+
+  let emulationUid = '';
+  let windowHandle: Window | null = null;
+  const startEmulation = async (uid: string) => {
+    const { emulationToken } = await fetch('/api/admin/emulationToken', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({ uid }),
+    }).then((res) => res.json());
+    windowHandle = window.open('', 'BlendEmulator', 'width=1920,height=1080,popup')!;
+    windowHandle.document.write(
+      `<iframe credentialless width="1920" height="1080" src="${`${PUBLIC_APP_URL}?jumpScene=${encodeURIComponent('res://Scenes/Account/Account.tscn')}&context=${encodeURIComponent(JSON.stringify({ token: emulationToken }))}`}" frameBorder="0" allowfullscreen></iframe>`,
```

Comment:
The `credentialless` parameter will make it so that the iframe has no access to existing cookies/local storage. Does not work on Safari.
https://developer.mozilla.org/en-US/docs/Web/Security/IFrame_credentialless